react-form-controls/my-app/src/RegistrationFormUncontrolled.js

```diff
@@ -0,0 +1,21 @@
+export default function RegistrationFormUncontrolled () {
+  function handleSubmit(e) {
+    e.preventDefault();
+    const $form = document.querySelector('#uncontrolled-form');
```

Comment:
Please get this without using `querySelector`. There is almost no case where you want to directly query the DOM with React.

Hint: the form is in event.target.